### PR TITLE
fix: address issue #40, increase modal z-index

### PR DESCRIPTION
**Pull request by BrentLabasan**
# Alaska Airlines Pull Request

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

**Fixes:** #40 

## Summary:

_Please summarize the scope of the changes you have submitted, what the intent of the work is, and anything that describes the before/after state of the project._

I increased the --auro-depth-modal value from 300 to 400, so that modals will appear over tooltips, which --auro-depth-tooltip is currently a value of 300.

This will fix situations like the picture below. This situation can be duplicated by clicking a non-saver fare in one matrix and then clicking a saver fare in the other matrix.
image

## Type of change:

Please delete options that are not relevant.

- [x] Revision of an existing capability

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

### CODE REVIEW

File: src/depth.json [hunk 1]
```diff
@@ -29,7 +32,7 @@
       "deprecated": false
     },
     "modal": {
-      "value": "{depth.300.value}",
+      "value": "{depth.400.value}",
```
Comment (blackfalcon) [inline src/depth.json:35]: I'd argue that `dialog` and `modal` are one in the same. So, please update both to be the same level and mark `modal` as deprecated. 

The primary token for use will be `dialog`. 

Dialog is a thing. Modal is a state.

**Review by blackfalcon — CHANGES_REQUESTED** [review]
See other notes about modal and dialog

File: src/depth.json (hunk 1 above)
Comment (blackfalcon) [reply src/depth.json:35]: @BrentLabasan have you had an opportunity to address this?

**Comment by BrentLabasan** [comment]
@blackfalcon  In AuroDesignTokens dialog's depth was updated to a value of 400, and modal was updated to be listed as deprecated.

**Comment by blackfalcon** [comment]
Closing PR in favor of https://github.com/AlaskaAirlines/AuroDesignTokens/pull/45

**Pull request closed without merging**